gits/test/helper.py

```diff
@@ -477,18 +477,22 @@ def _check_cap_il1(self, s, pos):
         # Restore the terminal to the sane modes.
         term._cap_rs1()
 
-    def _check_cap_kcub1(self, pos, want_cur_x):
+    def _check_cap_kcub1(self, pos):
         """A helper that checks the `_cap_kcub1` method.
 
-        The ``want_cur_x`` argument is an expected x position of the cursor
-        after calling the `cap_kcub1` method.
         The ``pos`` argument must be a tuple or list of coordinates ``(x, y)``
         of the initial position of the cursor.
         """
-        self._terminal._cur_x, self._terminal._cur_y = pos
-        self._terminal._cap_kcub1()
-        self.assertEqual(want_cur_x, self._terminal._cur_x)
-        self._terminal._cap_rs1()
+        term = self._terminal
+        x, _ = pos
+        term._cur_x, term._cur_y = pos
+
+        term._cap_kcub1()
+
+        self.assertFalse(term._eol)
+        self.assertEqual(x - 1, term._cur_x)
```

Comment:
We have to be able to check 

`self._check_cap_kcub1((0, 0))`

and make sure that `y` doesn't change after calling `_cap_kcub1`.
Howeve, in this case we will get `-1`.